bugsnag/client/api/base.ts

```diff
@@ -89,6 +107,19 @@ export class BaseAPI {
       apiResponse.body = results as T;
     }
 
+    this.sanitizeResponseURLs(apiResponse, this.configuration.basePath || 'https://api.bugsnag.com/');
```

Comment:
this is in the base API so will sanitize every single response from the API - I think that's too low a level as there could potentially be some situations where we might want to keep these fields, at least internally.

I think it may be better to do this at the error API level instead?

Reply:
the api url will be the same as the dashboard url for on-premise instances so this would remove any dashboard links as well. It would be better to exclude the api links from the response itself before its output by the tool. we should do that in the api client level